CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 - Deprecate the old `StringKey.discardAlertCancel` and `StringKey.discardAlertAction` string keys for overriding the discard alert buttons as they had incosistent behavior between iOS and Android ([#1001](https://github.com/Instabug/Instabug-React-Native/pull/1001)).
 - Deprecate the old `StringKey.reproStepsListItemNumberingTitle` string key for overriding the repro steps list item (screen) title as it had incosistent behavior between iOS and Android ([#1002](https://github.com/Instabug/Instabug-React-Native/pull/1002)).
 
+### Changed
+
+- Bump Insabug Android SDK to 11.14.0 ([#1019](https://github.com/Instabug/Instabug-React-Native/pull/1019)) [See release notes](https://github.com/Instabug/Instabug-Android/releases/tag/v11.14.0)
```

Comment:
There's a typo here in `Insabug`.

Reply:
Pardon me for missing this, and great catch as always 🔬